future/rsfmri-analysis/rsfmri-analysis.md

```diff
@@ -8,4 +8,124 @@ permalink: /docs/rsfmri-analysis
 
 # Resting State fMRI Analysis
 
+For resting state analysis, you can use XCP Imaging Pipeline. It's free and open-sourced.
```

Comment:
maybe briefly add the main features: various denoising strategies, outputs ALFF and ReHo (and others?)